slick-cats/src/test/scala/com/rms/miu/slickcats/DBIOInstancesTest.scala

```diff
@@ -9,7 +9,7 @@
 package com.rms.miu.slickcats
 
 import cats.data.EitherT
-import cats.implicits._
+import cats.syntax.all._
```

Comment:
Could you make it in alphabetic order, please.

Reply:
Done